Header header
float32 vel
float32[] activation_0
float32[] activation_1
float32[] activation_2
float32 distance